core/utils/silly-i18n/src/index.js

```diff
@@ -42,6 +42,27 @@ export class TranslationKeyError extends Error {
     }
 }
 
+/**
+ * Generates an object by slurping all JSON files
+ * @param sections - All different sections
+ * @param languages - Sections will come in these two different flavors
+ * @param path - Full path to import
+ * @returns an [[I18n]] object
+ */
+export async function createI18n(sections, languages, path) {
+    let i18nData = {};
+    for (const l of languages) {
+        i18nData[l] = {};
+    }
+    for (const s of sections) {
+        for (const l of languages) {
+            const section = await import(`${path}/${l}/${s}.json`);
```

Comment:
We had it like this originally, but I think it didn't work in some environments, either old Android WebView or the supported Safari version. Would be interesting to see if it works now. If it does, I think it'd be interesting to see if we can't maintain some API compatibility with i18next, by having an  `init` function.

If it doesn't work, one alternative would be to find a better way to bundle translation files - I believe i18next would actually need to use XHRs to access individual files (but I'm not sure) -> Currently we bundle everything into a single JS file for deployment.

Reply:
I've already checked with rollup, and it does not work, apparently, since dependencies are not explicit, so we might have to do a bit of magic there... But while we're at it, we might simply bundle all the JSON files into a single one, so there are many different paths. Anyway, the current way needs to be fixed somehow.
Also, if you can suggest some way to test that, I'd be inmensely grateful. Since this is using "import", there's no platform specific logic there so I guess we should be OK.

Reply:
These `import` statements are at the end of the day just using that Rollup JSON Plugin we're using. We could indeed look into what that plugin can do, or how to otherwise bundle a bunch of JSON files together to expose them as a JS module. Then we could handle the entire logic in Rollup. How exactly, I'd have to look into it.